lib/customer.rb

```diff
@@ -0,0 +1,38 @@
+require "csv"
+require "awesome_print"
+
+# Build Customer class, instantiate readers for instance variables.
+class Customer
+  attr_reader :id, :all_customers, :email, :address
```

Comment:
While I personally would make these all `attr_reader`s in my own code the spec did say to make everything but `id` read _and_ write.

It's not a big deal though, and I'm not sure if we maybe want to update the assignment.